### ISSUE TITLE
WAVE accessibility check: Form label needed

### ISSUE THREAD

**Issue by eff-soraya** [EFForg/starttls-frontend#168]
![screen shot 2018-06-13 at 9 58 52 am](https://user-images.githubusercontent.com/16709499/41366352-aa3734dc-6ef0-11e8-8679-b81a49b5581a.png)

It seems that the dropdown menu for "Do you run this email server?" is missing a form label.
